docs/ops/movement/BatchToSpace_2.md

```diff
@@ -4,46 +4,82 @@
 
 **Category**: *Data movement*
 
-**Short description**: The *BatchToSpace* operation reshapes the "batch" dimension 0 into N - 1 dimensions of shape `block_shape` + [batch] and interleaves these blocks back into the grid defined by the spatial dimensions `[1, ..., N - 1]` to obtain a result with the same rank as `data` input. The spatial dimensions of this intermediate result are then optionally cropped according to `crops_begin` and `crops_end` to produce the output. This is the reverse of the *SpaceToBatch* operation.
+**Short description**: The *BatchToSpace* operation reshapes the "batch" dimension 0 into N - 1 dimensions of shape `block_shape` + [batch] and interleaves these blocks back into the grid defined by the spatial dimensions `[1, ..., N - 1]` to obtain a result with the same rank as `data` input. The spatial dimensions of this intermediate result are then optionally cropped according to `crops_begin` and `crops_end` to produce the output.
```

Comment:
please split this long line sentences into into several lines for convenient reading in source.

Reply:
and the same comment for text below

Reply:
I have decided to simplify explanation in short description and include more details in the detailed one. The short description turned out to be copy-paste from tf spec which actually has different shapes as block_shape and crops inputs do not have the same elements as the rank of data input. It should be now more clear how this operation differs from tf BatchToSpaceND